samples/getting-started/qrng/Qrng.qs

```diff
@@ -16,18 +16,25 @@ namespace Qrng {
     }
     
     @EntryPoint()
-    operation Main() : Int {
+    operation RunMain() : Int {
```

Comment:
Since you're changing this anyway - what about `GenerateRandomNumber`?

Reply:
Sounds good, will do!

Reply:
Mmm, I like `Main()` here due to the nature of the operation, being executed by running the whole file. It resembles the conventions used in other languages.

Reply:
That's a good point, @geduardo. That said, in general, the Q# style guide suggests nouns should be reserved for functions and UDTs. While the style guide was drafted before the standalone feature, I'm inclined keep to that here as well, to go with @rmshaffer's suggestion of a more descriptive name. I think that while `Main` would be reflective of general-purpose languages, this reflects better that Q# programs act in a more accelerator-like model. Happy for further discussion on this, though!